spec.html

```diff
@@ -125,6 +125,61 @@ <h1>Updated Productions</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-new-ecmascript-specification-types">
+  <h1>New ECMAScript Specification Types</h1>
+
+  <emu-clause id="sec-classes-specification-types">
+    <h1>Classes Specification Types</h1>
+
+    <emu-clause id="sec-elementdescriptor-specification-type" aoid=ElementDescriptor>
+      <h1>The ElementDescriptor Specification Type</h1>
+
+      <p>
+        The ElementDescriptor is a Record used to represent class elements at runtime.
+        Values of the ElementDescriptor type are Record values whose fields are defined as by <emu-xref href="#table-1"></emu-xref>.
+      </p>
+
+      <emu-table id="table-1" caption="ElementDescriptor fields">
+        <table>
+          <thead>
+            <tr> <th>Field Name</th>      <th>Value</th>                                       </tr>
+          </thead>
+          <tbody>
+            <tr> <td>[[Kind]]</td>        <td>One of `"method"` or `"field"`</td>              </tr>
+            <tr> <td>[[Key]]</td>         <td>A Property Key or PrivateName</td>               </tr>
+            <tr> <td>[[Descriptor]]</td>  <td>A Property Descriptor</td>                       </tr>
+            <tr> <td>[[Placement]]</td>   <td>One of `"static"`, `"prototype"` or `"own"`</td> </tr>
+            <tr> <td>[[Initializer]]</td> <td>A function or ~empty~</td>                       </tr>
+            <tr> <td>[[Decorators]]</td>  <td>A List of functions or ~empty~</td>              </tr>
+          </tbody>
+        </table>
+      </emu-table>
+
+      <p>
+        In addition, given an ElementDescriptor _element_, the following conditions are always respected:
+        <ul>
+          <li>If _element_.[[Kind]] is `"method"`, then _element_.[[Initializer]] is ~empty~.</li>
```

Comment:
Perhaps it would be better to specify `[[Initializer]]` in terms of its absence or presence, to align with how Property Descriptors are defined. See [ToPropertyDescriptor](https://tc39.github.io/ecma262/#sec-topropertydescriptor), [IsAccessorDescriptor](https://tc39.github.io/ecma262/#sec-isaccessordescriptor), and [IsDataDescriptor](https://tc39.github.io/ecma262/#sec-isdatadescriptor) for reference.